### PR TITLE
Update to Zivid SDK v2.4.0

### PR DESCRIPTION
**Pull request by SatjaSivcev**

### CODE REVIEW

**Review by chrisasc — COMMENTED** [review]
If this is the only changes required to update to 2.4, it LGTM!

File: README.md [hunk 1]
```diff
@@ -48,22 +48,23 @@ There are two main categories of samples: **Camera** and **Applications**. The s
...
-2. [**Download Zivid Sample Data**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/450363393/Sample+Data).
+2. [**Download Zivid Sample Data**][zivid-sample-data-url].
 
 3. Launch Visual Studio 2017.
 
 4. Open and run one of the samples.
 
 ## Support
-If you need assistance with using Zivid cameras, visit our Knowledge Base at [help.zivid.com](https://help.zivid.com/) or contact us at [customersuccess@zivid.com](mailto:customersuccess@zivid.com).
+If you need assistance with using Zivid cameras, visit our [**Knowledge Base**][knowladge-base-url] or contact us at [customersuccess@zivid.com](mailto:customersuccess@zivid.com).
```
Comment (knzivid) [inline README.md:61]: typo in `knowladge-base-url`

**Review by knzivid — APPROVED** [review]
Remember to resolve open threads :)

**Comment by SatjaSivcev** [comment]
Thanks @knzivid. @chrisasc has to test all samples with cameras before we merge.

**Review by chrisasc — APPROVED** [review]
LGTM

**Pull request merged**